itest/test_list_on_test.go

```diff
@@ -5,6 +5,10 @@ package itest
 import "github.com/lightninglabs/taproot-assets/proof"
 
 var testCases = []*testCase{
+	{
+		name: "psbt stxo exclusion proofs",
+		test: testPsbtSTXOExclusionProofs,
```

Comment:
Don't see this new test added in this commit.

Reply:
Cherry-picking gone wrong.